Makefile

```diff
@@ -4,7 +4,7 @@
 
 .PHONY: build
 build:
-	cmake -B build
+	cmake -B build -DGGML_CUDA=$(GGML_CUDA) -DGGML_MUSA=$(GGML_MUSA)
```

Comment:
I don't think we have to update this target/recipe, at least my understanding is that using it should be somewhat discouraged and the preferred way is to use cmake directly.

Reply:
Yes, the `Makefile` is a temporary shim to make the transition to CMake easier.